README.md

```diff
@@ -221,61 +338,31 @@ Output of an operator after DKG is finished:
 	PartialSignature types.Signature
 ```
 
-## DKG protocol description
-
-#### Exchange message creation DKG protocol:
+### Note on DKG instance management
 
-1. Upon receiving init message from initiator, operator creates (if not exists for init msg ID[24]byte) a kyber-bls12381 instance consisting of
+A DKG-operator can handle multiple DKG instances, it saves up to `MaxInstances` (1024) up to `MaxInstanceTime` (5 minutes). If a new `Init` arrives the DKG-operator tries to clean instances older than `MaxInstanceTime` from the list. If any of them are found, they are removed and the incoming is added, otherwise it responds with an error, saying that the maximum number of instances is already running.
 
-- randomly generated scalar
-- corresponding point in elliptic curve group G1 (384 bit)
-
-2. Creates a signed with exchange message consisting of ID[24]byte and point bits
-
-#### DKG protocol steps at operator after receiving all exchange messages from the initiator
+---
 
-1. Generation of DKG nodes:
+## Security notes
 
-- operator ID uint64;
-- operators G1 point;
+It is important to briefly explain how the communication between DKG ceremony Initiator and Operators is secured:
 
-2. Creation of a time phaser
-3. DKG time phaser starts DealPhase
+1. Initiator is using RSA key (2048 bits) to sign `init` message sent to Operators. Upon receiving the signature, Operators verify it using public key included in the `init` message. If the signature is valid, Operators store this pub key for further verification of messages coming from the Initiator(s).
 
-- computes a private share for each of the operators ids
-- encrypts with a corresponding to the operator BLS public key created at exchange step
-- pack all deals together and signs
+2. Operators are using RSA key (ssv Operator key - 2048 bits) to sign every message sent back to Initiator.
 
-4. Deal bundle is created and sent back to the initiator
+3. Initiator verifies every incoming message from any Operator using ID and Public Key provided by Operators' info file, then Initiator creates a combined message and signs it.
 
-### DKG protocol steps at operator after receiving all deal messages from the initiator:
+4. Operators verify each of the messages from other Operators participating in the ceremony and verifies Initiator's signature of the combined message.
```

Comment:
if order is important here , we're verifying the combined message signed by the initiator first, then we verify each operator message with corresponding operator pubkey.